plugins/eos/gs-plugin-eos.c

```diff
@@ -1395,7 +1395,10 @@ app_is_banned_for_personality (GsPlugin *plugin, GsApp *app)
 	       ((g_strcmp0 (priv->personality, "zh_CN") == 0) &&
 	        (g_strv_contains (google_apps, app_name) ||
 	         g_str_has_prefix (app_name, "com.endlessm.encyclopedia"))) ||
-	       (g_str_has_prefix (priv->personality, "spark") &&
+               (g_str_has_prefix (priv->product_name, "spark") &&
```

Comment:
Nitpick: This line is over-indented by 1 space, which makes the nesting of the `&&`s and `||`s quite a lot harder to understand.